### PR TITLE
Fix issue when value has a "length" property. 

### PR DESCRIPTION
**Pull request by job13er**
### This project uses [semver](semver.org), please check the scope of this pr:
 - [ ] #none# - documentation fixes and/or test additions
 - [x] #patch# - backwards-compatible bug fix
 - [ ] #minor# - adding functionality in a backwards-compatible manner
 - [ ] #major# - incompatible API change

# CHANGELOG
 * **Fixed** [#85](https://github.com/ciena-blueplanet/bunsen-core/issues/85)
 * **Updated** `eslint-config-frost-standard` to `6.x` and fixed lint errors.

### CODE REVIEW

**Comment by job13er** [comment]
The majority of these changes are lint cleanup after upgrading to `eslint-config-frost-standard@6.x`

File: src/reducer.js [hunk 1]
```diff
@@ -68,8 +69,10 @@ function immutableOnce (object) {
  * @returns {Object} a value cleaned of any `null`s
  */
 function recursiveClean (value, model) {
-  let output = Array.isArray(value) ? [] : {}
-  _.forEach(value, (subValue, key) => {
+  let isValueArray = Array.isArray(value)
+  let output = isValueArray ? [] : {}
+  let iteratorFn = isValueArray ? _.forEach : _.forIn
+  iteratorFn(value, (subValue, key) => {
```
Comment (job13er) [inline src/reducer.js:75]: Here's the actual fix (using `_.forIn` when an object and `_.forEach` when an array.

File: tests/reducer-test.js [hunk 2]
```diff
@@ -147,6 +147,27 @@ describe('reducer', function () {
       expect(changedState.value).to.eql({baz: 22})
     })
 
+    it('can handle a value with a "length" property', function () {
```
Comment (job13er) [inline tests/reducer-test.js:150]: And here's the test that was failing before the fix.

**Comment by sandersky** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/ciena-blueplanet/bunsen-core/pull-request/86/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-blueplanet/bunsen-core)

**Pull request merged**